src/components/overlay-action/overlay-action.js

```diff
@@ -115,6 +118,60 @@ class OverlayAction extends BaseComponent {
     });
   }
 
+  /**
+   * Handler for overlay pointer (mouse/touch) down
+   *
+   * @param {*} event - mousedown/touchstart event
+   * @returns {void}
+   * @memberof OverlayAction
+   */
+  onOverlayPointerDown(event: any): void {
+    this._pointerDownPosX = event.clientX || event.changedTouches[0].clientX;
+    this._pointerDownPosY = event.clientY || event.changedTouches[0].clientY;
+  }
+
+  /**
+   * Handler for overlay mouse up
+   *
+   * @param {*} event - mouseup event
+   * @returns {void}
+   * @memberof OverlayAction
+   */
+  onOverlayMouseUp(event: any): void {
+    if (!this.props.isVr || !this.isDragging(event)) {
+      this.onOverlayClick();
+    }
+  }
+
+  /**
+   * handler for overlay touch end
+   *
+   * @param {*} event - touchend event
+   * @returns {void}
+   * @memberof OverlayAction
+   */
+  onOverlayTouchEnd(event: any): void {
+    if (this.props.playerHover && (!this.props.isVr || !this.isDragging(event))) {
+      this.togglePlayPause();
+    }
+  }
+
+  /**
+   * Whether the user is dragging
+   *
+   * @param {*} event - mouseup/touchend event
+   * @returns {boolean} - is dragging
+   */
+  isDragging(event: any): boolean {
+    if (event.clientX) {
```

Comment:
you can normalize the event to:
```javascript
const points = {
   clientX: event.clientX || event.changedTouches[0] && event.changedTouches[0].clientX,
   clientY: event.clientY || event.changedTouches[0] && event.changedTouches[0].clientY
}
return (points.clientX !== this._pointerDownPosX || points.clientY !== this._pointerDownPosY);
```

Reply:
@OrenMe  `event.changedTouches[0] && event.changedTouches[0].clientX` is not a boolean?